samples/metrics_monitor/test/igt_load.h

```diff
@@ -30,24 +30,17 @@
 #define __IGT_LOAD_H__
 
 #include <stdint.h>
+#include <stdbool.h>
```

Comment:
why you need it? very suspicious, could be implementation specific...

Reply:
Because of

/msdk/MediaSDK/samples/metrics_monitor/test/igt_load.h:41:1: error: unknown type name ‘bool’; did you mean ‘_Bool’?
 bool gem_has_bsd2(int fd);
 ^~~~
 _Bool

Reply:
fixed: this issue and failed build were caused by include "igt_load.h" position in igt_load.c